velox/type/Type.h

```diff
@@ -1300,6 +1300,22 @@ struct Array {
   Array() {}
 };
 
+// This is a temporary type to be used when ArrayProxy is requested by the user
+// to represent an Array output type in the simple function interface.
+// Eventually this will be removed and Array will be ArrayProxyT once all proxy
+// types are implemented.
+template <typename ELEMENT>
```

Comment:
Can you create an proxy for `ARRAY<VARBINARY>` using this?

Reply:
A following PR is adding support for string types (Varchar and varbinary).

Reply:
its not complete I meant to say .